### PR TITLE
security scan notice

### PR DESCRIPTION
**Pull request by jmazzitelli**
netlify: https://deploy-preview-673--kiali.netlify.app/news/security-bulletins/

### CODE REVIEW

File: content/en/news/security-bulletins/_index.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ type: docs
 weight: 2
 ---
 
+{{% alert color="info" %}}
+NOTE: Kiali takes security seriously and encourages users to report security concerns.
+
+If you run a security scan on Kiali software and would like to report a security scan report to the Kiali team, we only ask that you first verify that your scan is correctly validating the latest release and that the results are valid. Because the Kiali team takes security reports seriously, they often take priority over current work being done, and it takes the Kiali team a long time to research and validate them. So please make sure you verify that any reports you submit are reporting an accurate reflection of the Kiali software being scanned and that the security issues being reported actually affect Kiali or one of its dependencies.
```
Comment (jshaughn) [inline content/en/news/security-bulletins/_index.md]:
```suggestion
If you run a security scan on Kiali software and would like to report a security scan report to the Kiali team, we only ask that you first verify that your scan is correctly validating the latest release and that the results are valid.  Security report investigation often takes priority over scheduled work and can be time consuming for the Kiali maintainers to research and validate.  So, please verify that your submitted report accurately reflects the Kiali software being scanned, and that the reported security issue(s) actually affect Kiali or one of its dependencies.
```

**Review by jshaughn — COMMENTED** [review]
It's good, just a siggested edit.  Also, if you want this to go into effect immediately then we can backport to Current.

**Comment by jmazzitelli** [comment]
> It's good, just a siggested edit. Also, if you want this to go into effect immediately then we can backport to Current.

current PR: https://github.com/kiali/kiali.io/pull/674

**Review by jshaughn — APPROVED** [review]
Good!

**Pull request merged**